rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml

```diff
@@ -73,6 +73,7 @@
     - 17-rabbitmq.conf
     - 18-ceph.conf
     - 19-nginx.conf
+    - 98-traceback.conf
```

Comment:
The checks are failing with
```
fatal: [jrpcaioiad-5c1_logstash_container-c5f45846] => input file not found at /jenkins/workspace/RPC-AIO/rpcd/playbooks/roles/logstash/templates/98-traceback.conf or /jenkins/workspace/RPC-AIO/rpcd/playbooks/98-traceback.conf
```
It looks like the template wasn't added to the commit.